include/psp2kern/sblauthmgr.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * \kernelgroup{SceSblAuthMgr}
```

Comment:
Add the group there as well: https://github.com/vitasdk/vita-headers/blob/master/docs/definitions.dox